dockerbuild/idsync-cron

```diff
@@ -2,5 +2,5 @@
 # See https://crontab.guru/ for help with cron pattern.
 
 # m h dom mon dow user command
-1/2 * * * * root /data/run-full-batch.sh > /proc/$(cat /var/run/crond.pid)/fd/1 2>&1
-*/2 * * * * root /data/run-incremental-batch.sh > /proc/$(cat /var/run/crond.pid)/fd/1 2>&1
+2 0 * * * root /data/run-full-batch.sh > /proc/1/fd/1 2>&1
```

Comment:
Is the Process ID for cron in our Docker containers always `1`? Doing a quick test in a Docker container running on my computer, the `cat /var/run/crond.pid` command returned a `20`... I think.

Reply:
That's exactly the problem with using `$(cat /var/run/crond.pid)` to get the pid. We need to send to the stdout stream for process 1 because that's what goes to Docker logs. I fought with this for too long yesterday and then found a comment in that SO post you referenced the other day.

Reply:
Ohhhhh... excellent. I'm so glad you sorted through that to this solution, then. 👍